src/react-components/ui-root.js

```diff
@@ -499,33 +500,35 @@ class UIRoot extends Component {
   };
 
   render() {
-    if (!this.props.initialEnvironmentLoaded || !this.props.availableVREntryTypes || !this.props.janusRoomId) {
+    if (this.state.exited || this.props.roomUnavailableReason) {
+      const exitSubtitleId = `exit.subtitle.${this.state.exited ? "exited" : this.props.roomUnavailableReason}`;
+
       return (
         <IntlProvider locale={lang} messages={messages}>
-          <div className="loading-panel">
-            <div className="loader-wrap">
-              <div className="loader">
-                <div className="loader-center" />
-              </div>
-            </div>
+          <div className="exited-panel">
             <div className="loading-panel__title">
```

Comment:
Are these supposed to be `exited-panel` or are we just re-using the loading-panel styles? I think for proper BEM you should alias them in the css, but this is super minor.

Reply:
I wrote a separate bug for this, will deal with it at some point